keyboards/rart/rartpad/readme.md

```diff
@@ -0,0 +1,13 @@
+# [RARTPAD](https://github.com/alabahuy/RART/tree/master/RARTPAD)
+
```

Comment:
```suggestion
# [RARTPAD](https://github.com/alabahuy/RART/tree/master/RARTPAD)

![RARTPAD](https://user-images.githubusercontent.com/30220306/84770444-c2ed2a00-b001-11ea-9f7b-f532ec736083.png)
```

Reply:
Unless you have a better picture of the board.

Reply:
done